clustered/server/src/main/java/org/ehcache/clustered/server/management/Notification.java

```diff
@@ -22,5 +22,6 @@
   EHCACHE_SERVER_STORE_CREATED,
   EHCACHE_SERVER_STORE_ATTACHED,
   EHCACHE_SERVER_STORE_RELEASED,
-  EHCACHE_SERVER_STORE_DESTROYED
+  EHCACHE_SERVER_STORE_DESTROYED,
+  EHCACHE_SERVER_STORE_CLIENT_RECONNECTED
```

Comment:
@anthonydahanne @GaryWKeim  : FYI here are the notifications coming from Ehcache for multi-entity.